apiary.apib

```diff
@@ -825,6 +825,22 @@ Get a quotation
                 + type: `deal` (string)
                 + id: `53474a7a-f9b2-4dd4-88a8-40ce773c7a64` (string)
 
+## Tags [/tags]
+
+### tags.list [GET /tags.list]
+
+Get a list of tags.
+
++ Request (application/json;charset=utf-8)
```

Comment:
B : because we can never predict amounts and order in list calls, we have a _guideline_ to always provide paging and a default sorting method on list calls... can you add this?